api/bash/repo-list-export.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+#
+# set GITHUB_TOKEN to your GitHub or GHE access token
+# set GITHUB_API_ENDPOINT to your GHE API endpoint (defaults to https://api.github.com)
+
+if [ -n "$GITHUB_API_ENDPOINT" ]
+  then
+    url=$GITHUB_API_ENDPOINT
+  else
+    url="https://api.github.com"
+fi
+
+token=$GITHUB_TOKEN
+
+dependency_test()
+{
+  for dep in curl jq ; do
+    command -v $dep &>/dev/null || { echo -e "\n${_error}Error:${_reset} I require the ${_command}$dep${_reset} command but it's not installed.\n"; exit 1; }
+  done
+}
+
+token_test()
+{
+  if [ -n "$token" ]
+    then
+    token_cmd="Authorization: token $token"
+  else
+    echo "You must set a Personal Access Token to the GITHUB_TOKEN environment variable"
+    exit 1
+  fi
+}
+
+usage()
+{
+  echo -e "Usage: $0 [options] <orgname>...\n"
+  echo "Options:"
+  echo " -h | --help                      Help"
+  echo ""
+}
+
+get_repos()
+{
+  last_repo_page=$( curl -s --head -H "$token_cmd" "$url/orgs/$org/repos?per_page=100" | sed -nE 's/^Link:.*per_page=100.page=([0-9]+)>; rel="last".*/\1/p' )
```

Comment:
I'm using the `-E` flag here, which should [maintain compatibility between OS X/BSD & GNU sed](http://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed)

_/me shakes fist at incompatible UNIX commands_